stdlib/contrib/qxip/clickhouse/clickhouse.flux

```diff
@@ -0,0 +1,51 @@
+// Package query provides functions meant to simplify clickhouse http queries.
+//
+// The primary function in this package is `clickhouse.query()`
+//
+// ## Metadata
+// introduced: 0.187.1
+//
+package clickhouse
+
+import "csv"
+import "experimental"
+import "experimental/http/requests"
+
+// query queries data from a specified ClickHouse query with the provided parameters
+//
+// ## Parameters
+// - url: ClickHouse HTTP API.
+// - query: ClickHouse query to execute.
+// - limit: Query rows limit.
+// - max_bytes: Query bytes limit.
+// - format: Query format. Default CSVWithNames.
+//
+// ## Example
+// ```no_run
+// import "contrib/qxip/clickhouse"
+//
+// clickhouse.query(
+//   url: "https://play@play.clickhouse.com",
+//   query: "SELECT version()"
+// )
+// ```
+//
+// ## Metadata
+// tags: inputs
+//
+query = (
+    url="http://127.0.0.1:8123",
```

Comment:
I'm wondering if it would be better to define the default URL as a package option. So after your import statements, you'd add:

```js
option defaultURL = "http://127.0.0.1:8123"
```

And then update this line to:

```js
    url = defaultURL
```

So if a user wants to customize this URL, they would:

```js
import "contrib/qxip/clickhouse"

option clickhouse.defaultURL = "https://play@play.clickhouse.com"

clickhouse.query(query: "SELECT version()")
```

Reply:
This is actually a great suggestion. If this is the suggested way, we'd sure like to conform to the "house style" to fit well with other packages. And thank you SO much for the super valuable corrections, great learning experience for the next package.

Reply:
Implemented! Thanks again, I did not know options existed 👍